configure.scss

```diff
@@ -0,0 +1,53 @@
+$_n-ui-features: (
+	ads: false,
+	buttons: false,
+	grid: false,
+	colors: false,
+	icons: false,
+	typography: false,
+	cookieMessage: false,
```

Comment:
I'm on the fence as to whether the foundations/components distinction is worth having. Adds verbosity and, if we want to preserve the distinction at all levels of the API, complexity and demands on the user's memory.